src/__tests__/ansiCommandStream.test.js

```diff
@@ -11,35 +11,46 @@ describe('ansiCommandStream()', () => {
     jest.clearAllMocks();
   });
 
-  test('when yarn is installed use `yarn remove`', () => {
-    hasYarn.__setHasYarn(true);
-    ansiCommandStream({ args: ['remove', 'lodash'] });
-
-    expect(execa).toHaveBeenCalledTimes(1);
-    expect(execa).toHaveBeenCalledWith('yarn', ['remove', 'lodash']);
+  describe('with yarn installed', () => {
+    beforeAll(() => {
+      hasYarn.__setHasYarn(true);
+    });
+
+    test('use `yarn remove`', () => {
+      ansiCommandStream({ args: ['remove', 'lodash'] });
+
+      expect(execa).toHaveBeenCalledTimes(1);
+      expect(execa).toHaveBeenCalledWith('yarn', ['remove', 'lodash']);
+    });
+    test('use `yarn add`', () => {
+      ansiCommandStream({ args: ['add', 'lodash'] });
+
+      expect(execa).toHaveBeenCalledTimes(1);
+      expect(execa).toHaveBeenCalledWith('yarn', ['add', 'lodash']);
+    });
   });
 
-  test('when yarn is not installed use `npm uninstall`', () => {
-    hasYarn.__setHasYarn(false);
-    ansiCommandStream({ args: ['remove', 'lodash'] });
-
-    expect(execa).toHaveBeenCalledTimes(1);
-    expect(execa).toHaveBeenCalledWith('npm', ['uninstall', 'lodash']);
-  });
+  describe('when yarn is NOT installed', () => {
```

Comment:
It’s actually checking for yarn.lock in the folder, not if it’s installed, minor nit